jsx/Form.js

```diff
@@ -1752,11 +1752,15 @@ class LorisElement extends Component {
 class RadioElement extends React.Component {
   constructor(props) {
     super(props);
+    this.state = {
+      checked: this.props.checked,
+    };
     this.handleChange = this.handleChange.bind(this);
     this.generateLayout = this.generateLayout.bind(this);
   }
 
   handleChange(e) {
+    this.setState({checked: e.target.value});
```

Comment:
so this can be done in the parent component i.e. within handleRadio of the new_profile component. the parent component should be handling the state